voicevox_engine/model.py

```diff
@@ -113,13 +113,34 @@ class SpeakerStyle(BaseModel):
     id: int = Field(title="スタイルID")
 
 
+class SpeakerSupportSynthesisMorphing(str, Enum):
+    PERMIT = "PERMIT"
+    SELF_MORPHING_ONLY = "SELF_MORPHING_ONLY"
+    PROHIBIT = "PROHIBIT"
+
+    @classmethod
+    def _missing_(cls, value: object) -> "SpeakerSupportSynthesisMorphing":
+        return SpeakerSupportSynthesisMorphing.PERMIT
+
+
+class SpeakerSupportedFeatures(BaseModel):
+    """
+    スピーカーの対応機能
```

Comment:
（細かいですが。あと[`SupportedFeaturesInfo`](https://github.com/VOICEVOX/voicevox_engine/blob/master/voicevox_engine/model.py#L285)に表記を合わせて）
```suggestion
    話者の対応機能の情報
```